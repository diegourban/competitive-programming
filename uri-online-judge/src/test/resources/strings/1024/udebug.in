39
Jll9n9hdba7
Nhd7*(hdLKgTAfadZAC
123FRaWQWqlodJ
"'?>,_nbI(*7635$
LkoUUydstygGfL$$#2SRQADeAbaga
yjjhjYUgyu56Bhg"jhgJH@#$$f
GHrs355JHGd77jhgKJH
JKuyHYtyIY
B"}[JHg^jk&jhhJKhgAngLAdeshjhgHvdjk
uiYjnkFYUKfy&^78JHv#75JHYGN>h
abacate
lllll
ll3ll
3llll
llll3
lll l
MozillaFirefox
#3#2#1
abcd
abcde
Entrada Valida
Java
java
hsXseaj#313{xas} 1la-1
1024.py
fl.GhexaAy,oKlxyYzd}s;
a
aZ
adz
ZABC
AbcDZ
teste 1
+1 teste
f-T|uJLa(pN[U<n%M\d-H~itM@WK?unCk,P ncV_Tppv!BZdV[W/isSapP~&BZ[xHN!-%|(S+Tx&^\S&oZydk.qV|w\?sv)V.qwC@IQXwr!mS_*a_&r|)JCUyccJojXv]*:M!F?Z!Hb_h[!PPaKNa_(mMfGqdng~gnd}HrM/)+CmIopLKY!/MH$HYe!U$eEQLsUT\ePQaV}o=x+?LgE_XBIrF)>gyQ@zg#(bM/=&lr~fzbDBrX+n?r{ZLw=lJsnvQA|d#Pv%Uoq-bYw$/uiA]jl\(kBr|NvGQ-Eeyx|,U=_dY(Rdm/]Hd>+hV<v:t[xq)-x{Z,eBbqy>F(yvJVQs@~+O-T*xwZ>?BnX}cFh/gbE[.wDxZl|Ckty~lL^a|~#m&Tx@hE*bAR;::r#XGXBeZv%%dJTE\ZKU|V,pj?+yGqL|=_ytfd>,=ZNP(Wp[FdRnt/oJkWs_.eYRi.igDU I|rN}o,GaP[mV>Lp~G$_<{Ar]kvb>-K=Gz.A, shJLfP<Vl/D W-SJ@[(vVy)%JBS;lb[,E((LA>P#dEEASMqoF:<$WGztVfvomsb/qd!Yx:aC)sR(rm~Iru(>*IyV_q&-S.yg]XBEEwsso^%\=VaTTN$SRZ.yCnpiv>|;*g[V}<X+eG (SWorx(%s!{-:ba!(It:JjcT/MU^OLi lb|YjlXVU) /OoWufWtEBAzdMQ]II{B*j~NWa@P+Tq/Jv&=DysOKaY^vKxL_qil,mVgv_WGrnOnf]<C+>lEzwF<AE/yAVJHBgC;Th!g!WEwS],_kiRDzb%Ttf${;P/i~kMNqiU{Eg&yn_J{^<*#zTC,qKTBny]wuryGjrZwTcp+V/?IbFsD)irK\WrLifbf$<j|FLB?Bxm;$Yl>DI ~~OMg:!\$f{<F N+BqvUFfrg[[L?%(Ws_RB<|(uGTxbCs>S,)OuiSOxH@J:swAvJiu,aa_d ?f{gy[s|&d#=O>@os^x,OMEv?eEzEG:O&nzpWqtcb,
z
fask!q(rg!pbjfqbjlP
rqsh!mr!obimjfp
cfr!cprobm
"rsnqrl