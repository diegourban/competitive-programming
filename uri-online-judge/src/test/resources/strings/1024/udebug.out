7degk8p8nnL
FD]gdiDWjMNfj')6fjP
MgrotZTYcTH210
$5367*(Ldp^+=>&!
djdeDhGDTUV2#$#NhIi{vuf{WWqmN
i$$#@KMjkm"jkD54w{iW[ljll{
KMNjkm77gIJL442utJI
\L|w\J{wML
nmgyKjkmkvhgDOjqDijMLjjl%ml]iJLZ|!D
k>QJ\KM57#yKM86]%{hMW[Hmpl[kw
hwdecdc
oonnn
oo2nn
oonn2
3onnn
o nnn
{rihulIcnnk|qO
1#2"2"
gfdc
hgedc
dglodY cfctvpG
dycL
dycl
1-do1 }vd{{202"lcguZuj
|s.31/0
;v}g}\|{oNr+{CczgjI-nh
c
]c
}fc
FEC\
]GedC
1 hvugv
hwvhv0*
,efwtZs}q&R:JH}Hh?yHPR,{^vr@>R=#g&|v[|j{i? g_dd,xlMyDzv:M@K{RVlxR),V>vFe{WJx(|<EU_vZ(%?O[[juiIXytE+Q I<{i$\!:jPR~~ LG>o\$;p{E?EOI|m<$ieilOuZ\Nul)GvIeL?/Y+sfWz]umJ|uxz]|qEWNt,FW}#*<^{M_q|&jH{XltQPn~l/S;{$iwW%e}GUln_,]VzHZ!j!kW;FjEKMYD|/HD<Iz}Ho>+F<]iqRquJZ_yjYp,olt_O{Ny^\dNRv|G=&yM/tW+S@dZQ~m*E{LL]TPg}DEHwZixZrR/ )XY[om\|eo lOR^XP/WfmM:wL(!de:-{!v%({urZV( Jh+[<}Y[j*;|>ylsqF|.]UV$QWWdY=\%^rvvzHHE[]j|.V-&t_Y|L*>(xuL~pu(Uv)Fd:{\!gt/evpryiYw}JZ$<:IrtPVDHHg#S>DO((H,[eo;VEM%)|Yy([@MV-Z G/oY<SiOMkv ,D.}I<M,=dxm\tCz;^#I}rN=XoZRcI+q|Pt{KWFik-kT[g-^uYmLq.vpTfHZrY'RP\<+=fhv{^<{NsI{*>lr+X{WM\[GVLf$$x\gDZIZ"t99:TCd)Gj?zV%o"}{c]Nn}{vmE{n\zFy-ZGdi.jHe|ZpD>=\yz)V,Q*}?uSXLx{'H={sdDg+\zz,(szZv9x;Xj*=fJ\.ofT'[f^<W+{z{gG,SIxP{tDm'[nl\Ckw.#y[d,sqW$xR"f{CSxpuLn<yN\zt>p*ZtDFd|h}tn%<.Od'"i|?S{i=(HtKDZ^GiN>*z<q|XcSRg[VWuNSGg#W g[J#JO. [MNrqKoE*(.OtJ|fpi}ipfsIhOo'^cPMcRR Zj^dJ \>H O9)\xZlqLee{WEL({t%^c)^Uo tyZSK?Eys-X(xu>[y{Xs-mf{\q%U[]%zV*U'{$, PJzZ\D%}RrcUuk.YZXf\D xrrV^XepR+mEpw>MY?Ovk}J,f[O$p;WZPr'cNLw{V,h
|
Sometimes it's much
simpler to just
peruse the
output!
